src/test/SmartComponents/LandingPage/LandingPage.test.js

```diff
@@ -7,18 +7,14 @@ describe('Landing Page', () => {
     beforeEach(() => {
         renderWithReduxRouter(<LandingPage />);
     });
-    test('renders component', () => {
+    test('renders page heading', () => {
         // check heading
-        screen.getByRole('heading', { name: /image builder/i });
+        screen.getByRole('heading', { name: /images/i });
     });
-    test('renders CreateImageCard child component', () => {
-        // check CreateImageCard loads
-        screen.getByTestId('create-image-card');
-        // and has correct contents
-        screen.getByText(/create a new image/i);
-        screen.getByRole('button', { name: /create image/i });
-    });
-    test('renders ImagesCard child component', () => {
-        screen.getByText(/pending composes/i);
+    test('renders ImagesTable child component', () => {
+        // check action loads
+        screen.getByTestId('create-image-action');
```

Comment:
> Why was the test that checks the button removed?

I moved this check to this line. I really like the approach to use `.getByTestId` instead of other things, like `ByRole` or `ByText`, so I switched to that method here. One thing I like is that this uses `data-testid` on the element it's looking for, so it's obvious when working on UI updates which items are checked with testing, and it's easier to find where the item is defined in the tests.

Reply:
Oh right, yea that's fine then :)